teuthology/task/ceph_ansible.py

```diff
@@ -448,6 +451,82 @@ def run_rh_playbook(self):
         self.fix_keyring_permission()
         self.wait_for_ceph_health()
 
+    def run_haproxy(self):
+
+        """
+        task:
+            ceph-ansible:
+                haproxy: true
```

Comment:
can you add role definition that is needed like haproxy.i and haproxy.1

Reply:
I have already added in 'groups to roles' dict and the deployment was successful. Is it required elsewhere?

Reply:
Updated the doc string